components/compositing/Cargo.toml

```diff
@@ -31,6 +31,7 @@ net_traits = {path = "../net_traits"}
 num-traits = "0.2"
 pixels = {path = "../pixels", optional = true}
 profile_traits = {path = "../profile_traits"}
+rust-webvr = {version = "0.16", features = ["mock", "openvr", "vrexternal"]}
```

Comment:
is this related? There is another webvr change in another file.

Reply:
compositing depended on webvr for `VRServiceManager`. Moving `ConstellationMsg` to compositing made webvr depend on compositing, which created a cyclic dependency. To solve this, since `VRServiceManager` was just re-exported in webvr, I imported `VRServiceManager` directly from `rust-webvr` instead of `webvr`.

Reply:
Ok, good.